libminifi/test/Utils.h

```diff
@@ -115,6 +119,43 @@ void sendMessagesViaTCP(const std::vector<std::string_view>& contents, uint64_t
     tcp_message += '\n';
     socket.send(asio::buffer(tcp_message, tcp_message.size()), 0, err);
```

Comment:
similarly to `write_some`, `send` might also send less bytes than supplied

Reply:
Updated in 5d55facd7c7f853ed099d3bef89d0a0329b23043